st2common/st2common/models/api/execution.py

```diff
@@ -105,6 +105,22 @@ class ActionExecutionAPI(BaseAPI):
                 "type": "array",
                 "items": {"type": "string"},
                 "uniqueItems": True
+            },
+            "log": {
```

Comment:
At that point, I don't see what else could we add here. At the same time, I see no problem with expanding the log objects providing we keep in mind performance implications of transferring the execution down the wire.